poolqc/pooling_qc_sample.py

```diff
@@ -144,98 +134,67 @@ def update_title(lane_value, run_value):
     return 'You have selected lane {} in run {}'.format(lane_value, run_value)
 
 
-@app.callback(
-    [dep.Output('Summary Table', 'columns'),
-     dep.Output('Summary Table', 'data'),
-     dep.Output('Summary Table', 'style_data_conditional'),
-     dep.Output('download-link', 'href'),
-     dep.Output('download-link', 'download')],
-    [dep.Input('select_a_run', 'value'),
-     dep.Input('lane_select', 'value')]
-)
-def Summary_table(run_alias, lane_alias):
-    run = df[(df['Run'] == run_alias) & (df['LaneNumber'] == lane_alias)]
-    run = run[~run['Run'].isna()].drop_duplicates('library')
-    run = run[~run['library'].isna()]
+def Summary_table(run):
+    # Adding 'on-the-fly' metrics
     run['% Mapped to Coding'] = run['Coding Bases'] / run['Passed Filter Aligned Bases'] * 100
     run['% Mapped to Intronic'] = run['Intronic Bases'] / run['Passed Filter Aligned Bases'] * 100
     run['% Mapped to Intergenic'] = run['Intergenic Bases'] / run['Passed Filter Aligned Bases'] * 100
```

Comment:
The `% Mapped` variables are already present from ETL, no need to calculate again.